go.mod

```diff
@@ -10,11 +10,11 @@ require (
 	github.com/hashicorp/go-hclog v1.6.3
 	github.com/hashicorp/go-version v1.6.0 // indirect
 	github.com/turbot/go-kit v0.10.0-rc.0
-	//main
+	//main 7cb8edd1b25cfc85e9d5c56786fecf4f0ead9339 (fix: do not abort when unrelated connections are not yet loaded)
 	github.com/turbot/steampipe v1.7.0-rc.0.0.20240417105518-bd915693c2ea
```

Comment:
Hi  @kaidaguerre, shouldn't this have been updated to `github.com/turbot/steampipe v1.7.0-rc.0.0.20240606122218-7cb8edd1b25c`?

Reply:
yup quite right - this reference is incorrect